modules/islandora_iiif/config/schema/islandora_iiif.schema.yml

```diff
@@ -28,6 +28,9 @@ views.style.iiif_manifest:
     structured_text_term_uri:
       type: string
       label: "Structured text term"
+    getdimensions_from_sewrver:
+      type: boolean
+      label: "Retrieve image dimensions from IIIF server"
```

Comment:
Further looking, is such a value used anywhere?

Reply:
Good catch, removed.